Source/WebKit/WPEPlatform/wpe/WPEView.cpp

```diff
@@ -755,6 +755,46 @@ void wpe_view_unmap(WPEView* view)
     g_object_notify_by_pspec(G_OBJECT(view), sObjProperties[PROP_MAPPED]);
 }
 
+/**
+ * wpe_view_lock_pointer:
+ * @view: a #WPEView
+ *
+ * Disable the movements of the pointer in @view, locking it to a particular
+ * area; while the pointer is locked, mouse events are relative instead of
+ * absolute motions.
+ *
+ * Returns: %TRUE if view has support for pointer lock, or %FALSE otherwise
```

Comment:
We are also returning FALSE when already locked, so I would say something like %TRUE if pointer is locked, simialr to the unlock one